### PR TITLE
Make photos_albums_files.owner nullable for beta migrations

### PR DESCRIPTION
**Pull request by PVince81**
For https://github.com/nextcloud/photos/issues/1260 to unblock beta7

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

**Comment by PVince81** [comment]
or we ask users to purge that table if updating from another beta...

the table was indeed added recently in beta3: https://github.com/nextcloud/photos/commit/10f44aacc8f5161655694fff2dcca0f8d83ba3d1

File: lib/Migration/Version20001Date20220830131446.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 			$modified = true;
 			$table = $schema->getTable("photos_albums_files");
 			$table->addColumn('owner', Types::STRING, [
-				'notnull' => true,
+				'notnull' => false,
```
Comment (blizzz) [inline lib/Migration/Version20001Date20220830131446.php:72]:
```suggestion
				'notnull' => true,
				'default' => '',
```

**Review by blizzz — COMMENTED** [review]
setting an emptry string as default instead and keeping the not-null constraint

**Comment by PVince81** [comment]
yeah ok, let's go with empty string then @blizzz

**Comment by blizzz** [comment]
> yeah ok, let's go with empty string then @blizzz

Sorry, that was shit advice

`Column "oc_photos_albums_files"."owner" is NotNull, but has empty string or null as default.`

File: lib/Migration/Version20001Date20220830131446.php [hunk 2]
```diff
@@ -70,6 +70,7 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 			$table = $schema->getTable("photos_albums_files");
 			$table->addColumn('owner', Types::STRING, [
 				'notnull' => true,
```
Comment (nickvergessen) [inline lib/Migration/Version20001Date20220830131446.php:72]: Empty string is null in oracle 💥

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Pull request merged**